apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -176,7 +176,8 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                                 : "text-emerald-400"
                             } text-sm`}>
                             {slot.attendees ? seatsPerTimeSlot - slot.attendees : seatsPerTimeSlot} /{" "}
-                            {seatsPerTimeSlot} {t("seats_available")}
+                            {seatsPerTimeSlot}{" "}
+                            {t("seats_available", { count: seatsPerTimeSlot - slot.attendees })}
```

Comment:
This doesn't look like correct usage. there is no key seats_available

Reply:
the i18next plurals works with the key and count. 
it will automatically map the key with related values.

for example here. the key t("seats_available", {count:1}) will be mapped to 'seats_available_one'.
if you set the key t("seats_available", {count: number other than 1 }), it will be mapped to 'seats_available_other'

for more information check out, https://www.i18next.com/translation-function/plurals